main.js

```diff
@@ -0,0 +1,24 @@
+var menu = document.querySelector(".menu-mobile");
```

Comment:
...if you decided to use iife & DOMContentLoaded listener, let's use it here too.
Same with variable declaration - if you like the idea of using const/let (I believe it's worth using them, they are basically standard now), you could do the same thing here. There can be some conflicts after merging the branches (no matter which one goes first) and that's a great thing - you will anyways encounter many conflicts in the future and resolving them in this case will be easy. In case of any doubt -you know where to find me. :)